ci/terraform/account-management/manually-delete-account.tf

```diff
@@ -109,6 +109,22 @@ data "aws_iam_policy_document" "legacy_account_deletion_topic" {
   }
 }
 
+data "aws_iam_policy_document" "legacy_account_deletion_key" {
+  statement {
+    sid    = "KMSAccessForAccountDeletionSNS"
+    effect = "Allow"
+    actions = [
+      "kms:GenerateDataKey*",
+      "kms:Decrypt"
+    ]
+    resources = [var.legacy_account_deletion_topic_key_arn]
+  }
+}
+
+data "aws_iam_policy_document" "test" {
```

Comment:
is `test` deliberate?

Reply:
also I'm a bit rusty on these KMS policy documents but are you applying this to anything?

Reply:
> is test deliberate?

hah, no